src/main/java/frc/robot/Robot.java

```diff
@@ -128,21 +142,20 @@ public void autonomousPeriodic() {
 
   @Override
   public void teleopInit() {
-    if (teleopCommand != null) {
-      teleopCommand.schedule();
-    }
+
   }
 
   // use this to override the algorithm and just use a speed
   @Config
-  public void setOverrideSpeed(double overrideSpeed) {
+  public void setOverrideSpeed(final double overrideSpeed) {
     this.overrideSpeed = overrideSpeed;
     shooterSpeedController.setLaunchSpeed(this.overrideSpeed);
   }
 
   @Override
   public void teleopPeriodic() {
     CommandScheduler.getInstance().run();
+    disableCurvature();
```

Comment:
You should put the disableCurvature (or setDriveMode) call above the run call.

Reply:
done